update.py

```diff
@@ -43,8 +45,9 @@
 import argparse
 parser = argparse.ArgumentParser(description='Copy Common Files as needed, stripping out non-relevant wiki content')
 parser.add_argument('--site', help="If you just want to copy to one site, you can do this. Otherwise will be copied.")
-parser.add_argument('--clean', default='False', help="Does a very clean build - resets git to master head (and TBD cleans up any duplicates in the output).")
-parser.add_argument('--cached-parameter-files', default=False, help="Do not re-download parameter files", type=bool)
+parser.add_argument('--clean', action='store_true', help="Does a very clean build - resets git to master head (and TBD cleans up any duplicates in the output).")
+parser.add_argument('--cached-parameter-files', action='store_true', help="Do not re-download parameter files", default=False)
+parser.add_argument('--parallel', type=int, help="limit parallel builds, -1 for unlimited", default=-1)
```

Comment:
I'd prefer that this defaulted to 1, otherwise this will unexpectedly try to use all resources of the machine.

Reply:
in discussion with henry and randy we thought default parallel was better

Reply:
after looking at the amount of free memory on ardupilot.org I've changed my mind, and made it default to 1